README.md

```diff
@@ -16,12 +16,6 @@ To find the documentation for the latest Bootstrap 3 compatible release, check o
 
 If you would like to update React-Bootstrap within an existing project to use Bootstrap 4, please read our docs for **[migrating to React-Bootstrap V1](https://react-bootstrap.netlify.com/migrating/)**.
 
-## Related modules
-
-* [react-router-bootstrap][react-router-bootstrap] – Integration with [React Router][react-router]
-* [React Bootstrap Extended][react-bootstrap-extended] - A version of React Bootstrap where each component has extra props that map to bootstrap's utility classes. For example, `pullRight` adds the class `pull-right` to a component.
-* [Awesome React Bootstrap Components][awesome-react-bootstrap-components] - Additional components like off-canvas navbar, switch and sliders.
```

Comment:
this is probably still relevant no?

Reply:
it's still being maintained (last commit 8 days ago), so it seems relevant to this repository. I think we should still include it